pwiz_tools/Skyline/Model/Databinding/Entities/GlobalizedObjectResx.resx

```diff
@@ -0,0 +1,187 @@
+		that the ResXResourceWriter will generate, however the reader can 
+		read any of the formats listed below.
+
+		mimetype: application/x-microsoft.net.object.binary.base64
+		value   : The object must be serialized with 
+			: System.Serialization.Formatters.Binary.BinaryFormatter
+			: and then encoded with base64 encoding.
+
+		mimetype: application/x-microsoft.net.object.soap.base64
+		value   : The object must be serialized with 
+			: System.Runtime.Serialization.Formatters.Soap.SoapFormatter
+			: and then encoded with base64 encoding.
+
+		mimetype: application/x-microsoft.net.object.bytearray.base64
+		value   : The object must be serialized into a byte array 
+			: using a System.ComponentModel.TypeConverter
+			: and then encoded with base64 encoding.
+	-->
+  <xsd:schema id="root" xmlns="" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
+		<xsd:element name="root" msdata:IsDataSet="true">
+			<xsd:complexType>
+				<xsd:choice maxOccurs="unbounded">
+					<xsd:element name="data">
+						<xsd:complexType>
+							<xsd:sequence>
+								<xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+								<xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+							</xsd:sequence>
+							<xsd:attribute name="name" type="xsd:string" msdata:Ordinal="1" />
+							<xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+							<xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+						</xsd:complexType>
+					</xsd:element>
+					<xsd:element name="resheader">
+						<xsd:complexType>
+							<xsd:sequence>
+								<xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+							</xsd:sequence>
+							<xsd:attribute name="name" type="xsd:string" use="required" />
+						</xsd:complexType>
+					</xsd:element>
+				</xsd:choice>
+			</xsd:complexType>
+		</xsd:element>
+	</xsd:schema>
+  <resheader name="resmimetype">
+		<value>text/microsoft-resx</value>
+	</resheader>
+  <resheader name="version">
+		<value>1.3</value>
+	</resheader>
+  <resheader name="reader">
+		<value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=2.0.3500.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+	</resheader>
+  <resheader name="writer">
+		<value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=2.0.3500.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+	</resheader>
+  <data name="Category_AcquisitionInfo" xml:space="preserve">
+    <value>Measured</value>
+  </data>
+  <data name="Category_FileInfo" xml:space="preserve">
+    <value>File</value>
+  </data>
+  <data name="Category_MatchInfo" xml:space="preserve">
+    <value>Search</value>
+  </data>
+  <data name="Category_PrecursorInfo" xml:space="preserve">
+    <value>Precursor</value>
+  </data>
+  <data name="CCS" xml:space="preserve">
+    <value>CCS</value>
```

Comment:
Seems like a strange placement for spectrum properties text.

Reply:
This is where it was originally. Where should they be placed? These strings will be shared between 3 different forms.

Reply:
I think, on a SpectrumProperties.resx file. GlobalizedObject does not own these properties. It could be subclassed by any other object for which we want to display properties in the future. We want the property name text to live with the objects they describe not accumulate with the base class.

Reply:
That means that GlobalizedObject will have to look up the child class resource file at runtime. It's a bit clunky, IMHO, but doable.